server/events/group.js

```diff
@@ -108,15 +108,18 @@ export class GroupBatchHandler {
   updateUser(user, traits) {
     const diff = reduce(traits, (t, v, k) => {
       // drop nested properties
-      if (v !== user[`traits_group/${k}`] && typeof(v) !== "object") {
+      if (v !== user[`traits_group/${k}`] && typeof (v) !== "object") {
         t[`group/${k}`] = v;
       }
       return t;
     }, {});
 
     if (!isEmpty(diff)) {
       this.metric("updateUser");
-      return this.hull.as(user.id).traits(diff).then(() => {
+
+      const asUser = this.hull.asUser(user.id);
+      return asUser.traits(diff).then(() => {
+        asUser.logger.info("incoming.group.success", user);
         return { as: user.id, traits: diff };
       });
```

Comment:
@mkawalek how about `incoming.group.error` here for `then(() => {}, (error) => { /* here */ })`?